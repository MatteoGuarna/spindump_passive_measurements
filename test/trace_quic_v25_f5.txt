A test with F5 QUIC implementation server and Quant client.

There are four sessions:

- regular + full start
- regular + 0rtt
- key update
- quantum-mode

The commands used were:

   rm /tmp/quant*
   ./bin/client -i en0 https://f5quic.com:4433/15000 
   ./Bin/client -i en0 https://f5quic.com:4433/15000 
   ./bin/client -u -i en0 https://f5quic.com:4433/15000 
   ./bin/client -m -i en0 https://f5quic.com:4433/15000 

