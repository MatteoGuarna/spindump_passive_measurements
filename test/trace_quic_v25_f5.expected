QUIC 100.72.46.172 <-> 40.112.191.60 3a1d8103-e187ec26 (55005:4433) at 1580821535422894 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 3a1d8103-ff97150511e187 (55005:4433) at 1580821535582509 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 3a1d8103-ff97150511e187 (55005:4433) at 1580821535582509 measurement starting right 159615 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 null-ff97150511e187 (55005:4433) at 1580821535584864 measurement starting left 2355 packets 1 1 bytes 1228 68 bandwidth 1228 68 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 null-08ae883ea7b7223a (55005:4433) at 1580821535742631 change starting packets 2 1 bytes 2456 68 bandwidth 2456 68 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 82ad6284-08ae883ea7b7223a (55005:4433) at 1580821535744444 measurement starting left 1813 packets 2 2 bytes 2456 160 bandwidth 2456 160 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 82ad6284-084b25582fd7479e (55005:4433) at 1580821535903595 change starting packets 3 2 bytes 3684 160 bandwidth 3684 160 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 82ad6284-084b25582fd7479e (55005:4433) at 1580821535903595 measurement up right 159151 packets 3 2 bytes 3684 160 bandwidth 3684 160 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 82ad6284-084b25582fd7479e (55005:4433) at 1580821535903907 packet up responder length 1227 packets 3 4 bytes 3684 2614 bandwidth 3684 2614 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 82ad6284-084b25582fd7479e (55005:4433) at 1580821535904520 packet up responder length 929 packets 3 5 bytes 3684 3543 bandwidth 3684 3543 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 82ad6284-084b25582fd7479e (55005:4433) at 1580821535905766 packet up initiator length 104 packets 4 5 bytes 3788 3543 bandwidth 3788 3543 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 82ad6284-084b25582fd7479e (55005:4433) at 1580821535906157 packet up initiator length 74 packets 5 5 bytes 3862 3543 bandwidth 3862 3543 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 82ad6284-084b25582fd7479e (55005:4433) at 1580821535907323 packet up initiator length 1500 packets 6 5 bytes 5362 3543 bandwidth 5362 3543 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 82ad6284-084b25582fd7479e (55005:4433) at 1580821536065306 packet up responder length 424 packets 6 6 bytes 5362 3967 bandwidth 5362 3967 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 042d4659-d6b319096d1182415d52e5513cac38 (57604:4433) at 1580821536380251 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 042d4659-084f34d145045379 (57604:4433) at 1580821536539749 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 2d46596f-084f34d145045379 (57604:4433) at 1580821536542235 measurement starting left 2486 packets 1 1 bytes 1228 92 bandwidth 1228 92 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 2d46596f-08048b3cb9abf310 (57604:4433) at 1580821536705098 change starting packets 2 1 bytes 2456 92 bandwidth 2456 92 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 2d46596f-08048b3cb9abf310 (57604:4433) at 1580821536705098 measurement up right 162863 packets 2 1 bytes 2456 92 bandwidth 2456 92 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 2d46596f-08048b3cb9abf310 (57604:4433) at 1580821536707871 packet up initiator length 157 packets 3 3 bytes 2613 591 bandwidth 2613 591 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 2d46596f-08048b3cb9abf310 (57604:4433) at 1580821536717735 packet up initiator length 1500 packets 4 5 bytes 4113 878 bandwidth 4113 878 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 2d46596f-08048b3cb9abf310 (57604:4433) at 1580821536865176 packet up responder length 424 packets 4 6 bytes 4113 1302 bandwidth 4113 1302 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 047c03f2-2cf25c4608055c2af95307 (63416:4433) at 1580821537191774 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 047c03f2-0819b38b385c2427 (63416:4433) at 1580821537362557 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 7c03f2f7-0819b38b385c2427 (63416:4433) at 1580821537365387 measurement starting left 2830 packets 1 1 bytes 1228 92 bandwidth 1228 92 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 7c03f2f7-088a04fee7b49fd7 (63416:4433) at 1580821537522962 change starting packets 2 1 bytes 2456 92 bandwidth 2456 92 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 7c03f2f7-088a04fee7b49fd7 (63416:4433) at 1580821537522962 measurement up right 157575 packets 2 1 bytes 2456 92 bandwidth 2456 92 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 7c03f2f7-088a04fee7b49fd7 (63416:4433) at 1580821537525810 packet up initiator length 157 packets 3 3 bytes 2613 587 bandwidth 2613 587 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 7c03f2f7-088a04fee7b49fd7 (63416:4433) at 1580821537532675 packet up initiator length 1500 packets 4 5 bytes 4113 874 bandwidth 4113 874 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 7c03f2f7-088a04fee7b49fd7 (63416:4433) at 1580821537683047 packet up responder length 424 packets 4 6 bytes 4113 1298 bandwidth 4113 1298 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 044ea4bb-b6b94ac98cb33f164296 (63455:4433) at 1580821538024669 new starting packets 0 0 bytes 0 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 044ea4bb-b6b94ac98cb33f164296 (63455:4433) at 1580821538024718 packet starting initiator length 820 packets 2 0 bytes 2100 0 bandwidth 2100 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 044ea4bb-08ba0772ac431288 (63455:4433) at 1580821538183032 change starting packets 2 0 bytes 2100 0 bandwidth 2100 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 4ea4bbde-08ba0772ac431288 (63455:4433) at 1580821538185305 measurement starting left 2273 packets 2 1 bytes 2100 92 bandwidth 2100 92 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 4ea4bbde-08ba0772ac431288 (63455:4433) at 1580821538185307 measurement starting left 2275 packets 3 1 bytes 3380 92 bandwidth 3380 92 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 4ea4bbde-0847be9c7751dfe5 (63455:4433) at 1580821538343522 change starting packets 4 1 bytes 4258 92 bandwidth 4258 92 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 4ea4bbde-0847be9c7751dfe5 (63455:4433) at 1580821538343522 measurement up right 158215 packets 4 1 bytes 4258 92 bandwidth 4258 92 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 4ea4bbde-0847be9c7751dfe5 (63455:4433) at 1580821538346724 packet up initiator length 157 packets 5 5 bytes 4415 873 bandwidth 4415 873 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 4ea4bbde-0847be9c7751dfe5 (63455:4433) at 1580821538347419 packet up initiator length 1500 packets 6 5 bytes 5915 873 bandwidth 5915 873 note "V25,no spin"
QUIC 100.72.46.172 <-> 40.112.191.60 4ea4bbde-0847be9c7751dfe5 (63455:4433) at 1580821538503711 packet up responder length 424 packets 6 6 bytes 5915 1297 bandwidth 5915 1297 note "V25,no spin"
